app/src/main/res/layout/feeditem_fragment.xml

```diff
@@ -166,22 +166,22 @@
             </LinearLayout>
         </LinearLayout>
 
-        <TextView
-            android:id="@+id/noMediaLabel"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:visibility="gone"
-            android:textAlignment="center"
-            android:background="?android:attr/dividerVertical"
-            android:text="@string/no_media_label"/>
-
         <View
             android:layout_width="match_parent"
             android:layout_height="1dp"
             android:background="?android:attr/dividerVertical"/>
 
     </LinearLayout>
 
+    <TextView
+        android:id="@+id/noMediaLabel"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:visibility="gone"
+        android:textAlignment="center"
+        android:background="?android:attr/dividerVertical"
+        android:text="@string/no_media_label"/>
+
```

Comment:
Looks like something went wrong when rebasing/merging. Please revert this change